### PR TITLE
Instructions  for large use case setup

### PR DESCRIPTION
**Pull request by leclairm**

### CODE REVIEW

File: docs/models/icon/large_use_cases.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    end
+    subgraph "III) ICON-EXCLAIM"
+        direction LR
+        EST[Small Scale Test Case] ==> EIT[Intermediate Scale Test]
+        EIT                        ==> EFT[Full Scale Test]
+    end
+```
+
+## 1. Small Scale Test Case
+Set up an ICON test case (either ICON-c2sm or ICON-nwp) integrated in the ICON testing infrastructure with a low number of grid points and a few time steps. The idea here is to test the code path of the final setup and identify potential issues coming from upstream source code.
```
Comment (AnnikaLau) [inline docs/models/icon/large_use_cases.md:39]: what is meant by "(either ICON-c2sm or ICON-nwp)"?

Comment (mjaehn) [reply docs/models/icon/large_use_cases.md:39]: I think they should always set up a test case in `icon-nwp`. I think ICON-c2sm refers to our repo.

Comment (AnnikaLau) [reply docs/models/icon/large_use_cases.md:39]: I agree. I find this partly capitalization a bit confusing and we could link to icon-nwp here. What do you think?

Comment (leclairm) [reply docs/models/icon/large_use_cases.md]: > Do we maybe need to add something about getting access to icon-nwp also?

That was my concern. Not everybody has access to it. But, yes, we can consider these users will.

**Comment by AnnikaLau** [comment]
@mjaehn please take care of the rest

**Comment by mjaehn** [comment]
> @mjaehn please take care of the rest

Maybe @leclairm can help, too!

**Comment by AnnikaLau** [comment]
Deadline 20 November 2024:
@jonasjucker 2nd point
@leclairm 3rd point
@mjaehn  4th point
@jasper review

**Review by AnnikaLau — APPROVED** [review]

**Review by mjaehn — APPROVED** [review]
Very well done! 🐶

**Pull request merged**